<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CBookTextBrowser</class>
 <widget class="QWidget" name="CBookTextBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <widget class="MDocHeader" name="wdgDictHeader" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_9"/>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stwContent">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_1">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QToolButton" name="btHide">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string>hide panel</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="resfile.qrc">
                <normaloff>:/new/icons/icons/stop.png</normaloff>:/new/icons/icons/stop.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Preferred</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTabWidget" name="tabOpts">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="tabPosition">
             <enum>QTabWidget::North</enum>
            </property>
            <property name="tabShape">
             <enum>QTabWidget::Triangular</enum>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <property name="elideMode">
             <enum>Qt::ElideNone</enum>
            </property>
            <property name="documentMode">
             <bool>false</bool>
            </property>
            <widget class="QWidget" name="tab_3">
             <attribute name="icon">
              <iconset resource="resfile.qrc">
               <normaloff>:/new/icons/icons/loupe.png</normaloff>:/new/icons/icons/loupe.png</iconset>
             </attribute>
             <attribute name="title">
              <string>find</string>
             </attribute>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QToolButton" name="btHLText">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="toolTip">
                 <string>highlight text</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="resfile.qrc">
                  <normaloff>:/new/icons/icons/bulboff.png</normaloff>
                  <normalon>:/new/icons/icons/bulbon.png</normalon>:/new/icons/icons/bulboff.png</iconset>
                </property>
                <property name="shortcut">
                 <string>Meta+H</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="CLatCopt" name="txtFind" native="true">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_6">
                  <item>
                   <widget class="QComboBox" name="cmbType">
                    <item>
                     <property name="text">
                      <string>exact</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>regexp</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                  <item>
                   <widget class="QCheckBox" name="cbWordsOnly">
                    <property name="enabled">
                     <bool>true</bool>
                    </property>
                    <property name="toolTip">
                     <string>detect only words (separated by spaces)</string>
                    </property>
                    <property name="text">
                     <string>words</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_2">
                  <item>
                   <widget class="QToolButton" name="btTop">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="toolTip">
                     <string>top</string>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="icon">
                     <iconset resource="resfile.qrc">
                      <normaloff>:/new/icons/icons/uparrowto.png</normaloff>:/new/icons/icons/uparrowto.png</iconset>
                    </property>
                    <property name="shortcut">
                     <string>Meta+Home</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="btFindUp">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="toolTip">
                     <string>find prev</string>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="icon">
                     <iconset resource="resfile.qrc">
                      <normaloff>:/new/icons/icons/uparrow.png</normaloff>:/new/icons/icons/uparrow.png</iconset>
                    </property>
                    <property name="shortcut">
                     <string>Meta+PgUp</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="btFind">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="toolTip">
                     <string>find next</string>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="icon">
                     <iconset resource="resfile.qrc">
                      <normaloff>:/new/icons/icons/downarrow.png</normaloff>:/new/icons/icons/downarrow.png</iconset>
                    </property>
                    <property name="shortcut">
                     <string>Meta+PgDown</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="btBottom">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="toolTip">
                     <string>bottom</string>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="icon">
                     <iconset resource="resfile.qrc">
                      <normaloff>:/new/icons/icons/downarrowto.png</normaloff>:/new/icons/icons/downarrowto.png</iconset>
                    </property>
                    <property name="shortcut">
                     <string>Meta+End</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab">
             <attribute name="title">
              <string>content</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QCheckBox" name="cbRmAccents">
                <property name="text">
                 <string>remove non-word characters</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="cbRmSpaces">
                <property name="text">
                 <string>remove spaces</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_4">
             <attribute name="title">
              <string>appearance</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QCheckBox" name="cbAllowFont">
                  <property name="text">
                   <string>allow change</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>47</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QFontComboBox" name="cmbF"/>
                </item>
                <item>
                 <widget class="QSpinBox" name="spnFSize">
                  <property name="minimum">
                   <number>10</number>
                  </property>
                  <property name="value">
                   <number>20</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="br">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="openLinks">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="wdgInfo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <property name="spacing">
           <number>12</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>20</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>lines:</string>
            </property>
            <property name="textFormat">
             <enum>Qt::PlainText</enum>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <property name="indent">
             <number>0</number>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::NoTextInteraction</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblInf">
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="text">
             <string notr="true">?</string>
            </property>
            <property name="indent">
             <number>-1</number>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::NoTextInteraction</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>chars:</string>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::NoTextInteraction</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblChar">
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="text">
             <string notr="true">?</string>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::NoTextInteraction</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbWrap">
            <property name="text">
             <string>wrap</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btSaveAs">
            <property name="toolTip">
             <string>save as</string>
            </property>
            <property name="text">
             <string>save as</string>
            </property>
            <property name="icon">
             <iconset resource="resfile.qrc">
              <normaloff>:/new/icons/icons/save.png</normaloff>:/new/icons/icons/save.png</iconset>
            </property>
            <property name="shortcut">
             <string>Ctrl+S</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>284</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="btAction">
            <property name="text">
             <string>action</string>
            </property>
            <property name="icon">
             <iconset resource="resfile.qrc">
              <normaloff>:/new/icons/icons/action.png</normaloff>:/new/icons/icons/action.png</iconset>
            </property>
            <property name="shortcut">
             <string>Meta+A</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="0" margin="0"/>
 <customwidgets>
  <customwidget>
   <class>CLatCopt</class>
   <extends>QWidget</extends>
   <header>latcopt.h</header>
   <container>1</container>
   <slots>
    <signal>query()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>MDocHeader</class>
   <extends>QWidget</extends>
   <header>mdocheader.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabOpts</tabstop>
  <tabstop>br</tabstop>
  <tabstop>cbWrap</tabstop>
  <tabstop>btSaveAs</tabstop>
  <tabstop>btAction</tabstop>
  <tabstop>btHide</tabstop>
  <tabstop>btHLText</tabstop>
  <tabstop>cmbType</tabstop>
  <tabstop>cbWordsOnly</tabstop>
  <tabstop>btTop</tabstop>
  <tabstop>btFindUp</tabstop>
  <tabstop>btFind</tabstop>
  <tabstop>btBottom</tabstop>
  <tabstop>cbRmAccents</tabstop>
  <tabstop>cbRmSpaces</tabstop>
  <tabstop>cbAllowFont</tabstop>
  <tabstop>cmbF</tabstop>
  <tabstop>spnFSize</tabstop>
 </tabstops>
 <resources>
  <include location="resfile.qrc"/>
 </resources>
 <connections/>
</ui>
